docs/OV_Runtime_UG/Integrate_with_customer_application.md

```diff
@@ -66,235 +61,124 @@ Use the following code to create Inference Engine Core to manage available devic
-   
-   5. **Run on multiple images** with setting batch. If you want to run inference for multiple images at once, you can use the built-in batch pre-processing functionality.
-   
-      **NOTE** : Batch pre-processing is not supported if input color format is set to `ColorFormat::NV12`.
+2. Request input and output information using `ov::Model::inputs()`, and `ov::Model::outputs()` methods:
+    @snippet snippets/src/main.cpp part2
+    @snippet snippets/src/main.cpp part3
+   To apply some pre-post processing please read this article about [OpenVINO™ Runtime PrePostProcessor].
 
 @sphinxdirective
 .. raw:: html
 
     </div>
 @endsphinxdirective
 
-#### Step 3. Load the Model to the Device
-
-Load the model to the device using `InferenceEngine::Core::LoadNetwork()`:
-
-
-@sphinxdirective
-   
-.. tab:: IR
-
-   .. code-block:: c
-
-      executable_network = core.LoadNetwork("model.xml", "CPU");
-
-.. tab:: ONNX
-
-   .. code-block:: c
-
-      executable_network = core.LoadNetwork("model.onnx", "CPU");
-
-.. tab:: nGraph
+#### Step 3. Compile the Model
 
-   .. code-block:: c
+Compile the model to the device using `ov::Core::compile_model()`:
 
-      std::shared_ptr<Function> createNetwork() {
-         // To construct a network, please follow 
-         // https://docs.openvino.ai/latest/openvino_docs_nGraph_DG_build_function.html
-      }
-      auto network = CNNNetwork(createNetwork());
-      executable_network = core.LoadNetwork(network, "CPU");
-
-.. tab:: Model From Step 2
-   
-   Follow this step only if you went through optional "Step 2 (Optional). Configure Input and Output of the Model", otherwise use another tab for your model type: IR (OpenVINO Intermediate Representation), ONNX or nGraph.
-   
-   .. code-block:: c
+   - IR:
+        @snippet snippets/src/main.cpp part4_1
+   - ONNX:
+        @snippet snippets/src/main.cpp part4_2
+   - Paddle:
+        @snippet snippets/src/main.cpp part4_3
+   - OpenVINO Model:
+        @snippet snippets/src/main.cpp part4_4
 
-      executable_network = core.LoadNetwork(network, "CPU");
 
-@endsphinxdirective
-
-
-It creates an executable network from a network object. The executable network is associated with single hardware device.
+It creates a compiled model from a model object. The compiled model is associated with single hardware device.
 It is possible to create as many networks as needed and to use them simultaneously (up to the limitation of the hardware resources).
 
 Third parameter is a configuration for plugin. It is map of pairs: (parameter name, parameter value). Choose device from
```

Comment:
- for plugin => for device
- It is map of pairs: (parameter name, parameter value) => list of properties which affects device behavior